fetchit_challenge/launch/fetchit_challenge_arena_montreal2019.launch

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<launch>
+
+  <arg name="world_name" default="$(find fetchit_challenge)/worlds/fetchit_challenge_arena_montreal2019.world"/>
```

Comment:
@RDaneelOlivav I'm not a fan of the growing number of worlds and launch files in this repository but at the same time, I want o merge this now and then clean things up later.

Reply:
Yeah, you're right. I think we should open an Issue about this and clean up a bit to only leave the necessary launches and worlds, Otherwise its too cluttered.